core/designsystem/src/main/kotlin/com/dhc/designsystem/calendar/ui/DhcCalendarDate.kt

```diff
@@ -41,14 +33,19 @@ fun DhcCalendarDateSwiper(
         state = pagerState,
         pageSpacing = 8.dp,
     ) { page ->
-        DhcCalendarDate(day = controller.getDateByPage(page))
+        DhcCalendarDate(
+            day = controller.getDateByPage(page),
+            monthData = controller.calendarMonthState[controller.getDateByPage(page)]
+                ?: DhcCalendarMonthData(yearMonth = controller.getDateByPage(page)),
+        )
```

Comment:
```suggestion
        val date = controller.getDateByPage(page)
        DhcCalendarDate(
            day = date,
            monthData = controller.calendarMonthState[date]
                ?: DhcCalendarMonthData(yearMonth = date),
        )
```
P2 : 여기도 함수 중복호출되는 부분을 모아줄 수 있을 것 같아용

Reply:
@JeonK1 
음 좋군 ...